tidb-cloud/terraform/terraform-overview.md

```diff
@@ -0,0 +1,43 @@
+---
+title: Overview
+summary: Create, manage, and update your TiDB Cloud resources through Terraform
+---
+
+# Overview
+
+TiDB Cloud already automates many database operations, and with our [Open API](https://docs.pingcap.com/tidbcloud/api/v1beta) you can programmatically handle additional operations like deployment, scaling, and restore.
+
+If you are looking to a simplified ways to automate resource provisioning and your infrastructure workflow. You can try out our TiDB Cloud Terraform provider with the capabilities:
+
+- Get your project information
+- Get cluster spec information, like supported cloud_provider, region, node size and so on
+- Mange your developer tier and dedicated tier, including create, scale, paused and resume the cluster
+- Create and delete the backup for your cluster
+- Create the restore task for your cluster
+
+## Requirements
+
+- [TiDB Cloud](https://docs.pingcap.com/tidbcloud/release-notes): compatible with TiDB Cloud release 20220927
```

Comment:
I think `https://docs.pingcap.com/tidbcloud/release-notes` will redirect to the newest release-notes. if we use release-notes-2022.md, it need to be updated per year.

and we just compatible with September 27, 2022.  not since